### ISSUE TITLE
 The issue of 'Error connecting to the service protocol' reappears 

### ISSUE THREAD

**Issue by AnonymHK** [zezo357/flutter_meedu_videoplayer#166]
I deleted the build directory and encountered an error after regeneration! 

FAILURE: Build failed with an exception.

* Where:
  Build file 'xx\media_kit_libs_android_video-1.3.0\android\build.gradle' line: 80

* What went wrong:
A problem occurred evaluating project ':media_kit_libs_android_video'.
> https://github.com/media-kit/libmpv-android-video-build/releases/download/v1.1.0/default-armeabi-v7a.jar

 I checked the build. gradle file and found that the author of "libmpv android video build" removed all releases and only retained the latest v1.1.2 version, resulting in errors in all compilations. I think he is crazy....

**Comment by zezo357** [comment]
Well well , it's like he wants to just kill any workaround until all people just use his package and don't care about anyone else using another logic around it🥹

**Comment by zezo357** [comment]
Anyway will see what I can do before work, I don't think it's logical to wait to the weekend in that case, since it breaks my workaround

**Comment by zezo357** [comment]
this is a work in progress, but i think it should work on all platforms only web is not supported for now 
```
  flutter_meedu_videoplayer:
    git:
      url: https://github.com/zezo357/flutter_meedu_videoplayer
      ref: fvp
      path: ./package
```

let me know if you have a problem with it, for now it will use fvp in all platforms

**Comment by AnonymHK** [comment]
> this is a work in progress, but i think it should work on all platforms only web is not supported for now
> 
> ```
>   flutter_meedu_videoplayer:
>     git:
>       url: https://github.com/zezo357/flutter_meedu_videoplayer
>       ref: fvp
>       path: ./package
> ```
> 
> let me know if you have a problem with it, for now it will use fvp in all platforms

 It seems that there is currently no other solution besides waiting for your update.

**Comment by zezo357** [comment]
> > this is a work in progress, but i think it should work on all platforms only web is not supported for now
> > ```
> >   flutter_meedu_videoplayer:
> >     git:
> >       url: https://github.com/zezo357/flutter_meedu_videoplayer
> >       ref: fvp
> >       path: ./package
> > ```
> > 
> > 
> >     
> >       
> >     
> > 
> >       
> >     
> > 
> >     
> >   
> > let me know if you have a problem with it, for now it will use fvp in all platforms
> 
> It seems that there is currently no other solution besides waiting for your update.

didnt that work for you? dont add it as an override, just as a package?

**Comment by AnonymHK** [comment]
Obviously, I am completely unable to use this method when debugging examples and fixing bugs! This method is meaningless for fixing bugs, and the production environment will never be updated until this issue is resolved! So now it's very awkward, I seem to have nothing to do anymore!

**Comment by AnonymHK** [comment]
This solution has no meaning for debugging, as media_ The author of the kit has removed the old library files, and I cannot even successfully run 'flutter_meedu_videoplayer' now!

**Comment by zezo357** [comment]
yes i get it this why i switched to another library which is fvp. but the code is not final there yet so i cant release it 

i did make it work, but i have to get sometime to be able to release it

**Comment by AnonymHK** [comment]
How long is it expected to take for the release of Fvp? I can't do anything now, it's quite awkward .....

**Comment by zezo357** [comment]
i am frustrated like you, but i really have to finish work before i do anything,

i will try my best to release it today

**Comment by AnonymHK** [comment]
May I ask a question? When debugging, I need to display a progress bar by default! The file is located in "lib src widgets styles primary primary_player_controls. dart". There is a call to PrimaryBottomControls under this PrimaryVideoPlayerControls function that is related to the progress bar, but I am currently encountering a problem. PrimaryVideoPlayerControls called build, but the build under PrimaryBottomControls was not called. I tried printing and other methods, but there were no errors, But it just doesn't call... Can you help check it out？

**Comment by zezo357** [comment]
please check new release
